components/BillHistoryTable/BillHistoryTable.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { Table } from 'react-bootstrap'
+
+const BillHistoryTable = (props) => {
+	const { documentHistoryActions } = props
+
+	const DocumentHistoryActionRows = () => {
```

Comment:
Components should always be defined at the top level -- as it is this component is redefined each render. We should move this outside the BillHistoryTable, so it's defined at the module level, and then pass the props to it and have it access the documentHistoryActions from the props.